esf_main.tex

```diff
@@ -30,10 +30,20 @@ \subsubsection{Wiring, cables, current calculations, connectors}
 	\includegraphics[width=\linewidth]{FormulasForDistanceAndSoundLevel}
 	\caption{Inverse Square Law equations needed to calculate the volume at a certain distance}
 \end{figure}
-Using the inverse square law for sound pressure levels, 97db at 122cm will translate to 93db at 2m which is a loud enough volume without inducing harm to the listener. 
+Using the inverse square law as referenced in the appendix  for sound pressure levels \ref{Ready to Drive}, 97db at 122cm will translate to 93db at 2m which is a loud enough volume without inducing harm to the listener. 
 \subsubsection{Position in car}
 %Provide CAD-renderings showing all relevant parts. Mark the parts in the rendering, if necessary.
 The ready to drive sound will be located in the dashboard enclosure shown in Figure !!!!. The buzzer will be mounted
 to the exterior and front side of this enclosure. It must be contained outside of the box so that the buzzer is
 loud enough. The speaker is waterproof, so it doesn't require any extra protection besides using an RTV silicone seal for final instillation. 
+
+
+\newpage
+\subsection{Ready to Drive (APPENDIX SECTION)}\label{Ready to Drive}
```

Comment:
Can you put this as a subsection after Section 11 (Appendix)? Sorry to keep running this back and forth.